src/libs/workspace-utils.ts

```diff
@@ -71,7 +71,16 @@ export interface WorkspaceSubmissionStats {
   runningSubmissionsCount: number;
 }
 
-export type WorkpaceState =
+export interface WorkspaceAccessEntry {
+  accessLevel: WorkspaceAccessLevel;
+  pending: boolean;
```

Comment:
I'm not familiar with `pending`.

Reply:
removed the type in favor of an existing one, but FWIW, I just pulled it straight out of swagger.